src/protocol-reference.rst

```diff
@@ -713,6 +713,23 @@ Streaming instructions
     :arg string filename:
         The name of the file, as it would be saved on a filesystem.
 
+.. guac:instruction:: msg
+    :sent-by: server
+    :phase: interactive
+
+    Sends a message from the server (gaucd) to the client. The nature of these
+    messages is intentionally ambiguous and flexible - the message must include
+    a numeric code that the client understands and can act on, and may also
+    include one or more arguments that can be used by the client in association
+    with the message.
+
+    :arg integer msg:
+        A numeric value indicating the message that is being passed to the client.
```

Comment:
We should document those values, as well, similar to: https://guacamole.apache.org/doc/gug/protocol-reference.html#status-codes

Reply:
Okay, will add them after the status codes.

Reply:
Added a commit with the message code documentation.